src/containers/eventRSVP/index.tsx

```diff
@@ -0,0 +1,140 @@
+import { Alert, Space, Spin, Table, Typography } from 'antd';
+import React, { useEffect } from 'react';
+import { Helmet } from 'react-helmet';
+import { connect, useDispatch } from 'react-redux';
+import { Link as RouterLink, useParams } from 'react-router-dom';
+import { ChungusContentContainer } from '../../components';
+import { C4CState } from '../../store';
+import {
+  asyncRequestIsComplete,
+  asyncRequestIsFailed,
+  asyncRequestIsLoading,
+} from '../../utils/asyncRequest';
+import { getUpcomingEvents } from '../upcoming-events/ducks/thunks';
+import { EventsReducerState } from '../upcoming-events/ducks/types';
+import { getEventRegistrations } from './ducks/thunks';
+import { EventRegistrationsReducerState, Registration } from './ducks/types';
+
+const { Title, Link } = Typography;
+interface EventRSVPProps {
+  readonly events: EventsReducerState['upcomingEvents'];
+  readonly registrations: EventRegistrationsReducerState['eventRegistrations'];
+}
+
+const TablePhotoConsent = (text: Registration, record: Registration) => (
+  <Space size="middle">{record.photoRelease ? 'Yes' : 'No'}</Space>
+);
+const TableActions = (text: Registration, record: Registration) => (
```

Comment:
: React.FC<>?

Reply:
Hmm not quite because these are actual function args and not contained in the props, I think its ok to leave this untyped because this is pretty AntD specific render function